Source/UI/BillingForm/BillingFormCell.swift

```diff
@@ -51,6 +51,19 @@ import Foundation
         }
     }
 
+    var accessibilityIdentifier: String {
+        switch self {
+        case .fullName: return AccessibilityIdentifiers.BillingForm.cardholder
+        case .addressLine1: return AccessibilityIdentifiers.BillingForm.addressLine1
+        case .addressLine2: return AccessibilityIdentifiers.BillingForm.addressLine2
+        case .city: return AccessibilityIdentifiers.BillingForm.city
+        case .state: return AccessibilityIdentifiers.BillingForm.state
+        case .postcode: return AccessibilityIdentifiers.BillingForm.postcode
+        case .country: return ""
+        case .phoneNumber: return AccessibilityIdentifiers.BillingForm.phoneNumber
+        }
+    }
```

Comment:
I think a typealias can come in handy here

Reply:
I don't understand, can you please provide an example? 
And where would it be used, just within `BillingFormCell`, or around the `AccessibilityIdentifiers` enum?

Reply:
I mean like `typealias Identifier = AccessibilityIdentifiers.BillingForm`